herddb-core/src/test/java/herddb/server/ClientMultiServerTest.java

```diff
@@ -225,6 +225,11 @@ public void requestMetadataRefresh(Exception error) throws ClientSideMetadataPro
                     } catch (ClientSideMetadataProviderException ok) {
                          assertTrue(ok.getCause() instanceof LeaderChangedException);
                     }
+
+                    // ensure that we don't use internal plans cache
```

Comment:
without this fix the test passed on Calcite, but not with jSQLParser

Reply:
:O